### DISCUSSION TITLE
Proposal to move Fil+ governance call from bi-weekly to monthly cadence

### DISCUSSION THREAD

**Discussion by raghavrmadya** [filecoin-project/notary-governance#612] [Polls]
**Motivation**: Over the last year, it has been great to see more and more community members join the bi-weekly Fil+ governance calls and actively engage with issues.  As interest and attendance in the governance calls has grown, it has become difficult to cover the various topics ranging from trust and safety, enterprise and encrypted data, FIPs etc. in the one-hour block allocated to the governance calls. Today, topics discussed shared in governance calls are inconsistent and pressing issues that are tactical in nature tend to dominate the calls preventing strategic discussions relating to the Fil+ program.

**The proposal**
Move the governance calls to a monthly cadence where 2/3 of the call is dedicated for the following Working Groups (WGs) present updates and 1/3 of the time is spent on discussing broader governance-focused issues.

Currently, the following WG have been stood up: 
1. E-Fil WG led by @kevzak
2. Trust & Transparency WG led by @raghavrmadya 

The following WG are planned to be initiated in the coming weeks:
1. SP WG led by TBD
2. Notary WG led by @Kevin-FF-USA (we already have a Notary focused channel FWIW)
3. Client Onboarding WG led by TBD

The proposed model of engagement for the governance calls will increase efficacy of the calls and help channel the ideas, energy, and passion of the community providing dedicated space(s) to brief and de-brief

**Comment by jamerduhgamer** [comment]
I agree with this change to have it be a monthly cadence instead. The way I see it is that the FIL+ governance meeting has already been broken up into the other WGs outlined in the proposal.

The monthly meeting can serve to be a summary of the different WGs / discuss important issues that may have come up since the previous meeting.
